charts/jaeger/templates/query-deploy.yaml

```diff
@@ -44,6 +44,10 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- if .Values.query.initContainers }}
```

Comment:
we should add default value in values.yaml file
`initContainers: []`
similar to: https://github.com/jaegertracing/helm-charts/blob/main/charts/jaeger/values.yaml#L521